.prettierignore

```diff
@@ -61,3 +61,5 @@ testCourse/**/*.html
 
 # JSON files in tests are formatted explicitly to check for loading behaviour, so should not be formatted automatically
 apps/prairielearn/src/tests/*/**/*.json
+
+.venv
```

Comment:
Nit, seems unrelated to this PR? I'd rather save this for #11033.

Reply:
Fixed in a20921b23.